qfieldsync/core/cloud_transferrer.py

```diff
@@ -193,19 +220,23 @@ def _delete(self) -> None:
                 if project_file.checkout & ProjectFileCheckout.Local:
                     Path(project_file.local_path).unlink()
 
-                reply = self.network_manager.delete_file(
-                    self.cloud_project.id + "/" + str(filename) + "/"
-                )
-                reply.finished.connect(
-                    self._on_delete_file_finished_wrapper(reply, filename)
-                )
+        self.throttled_deleter.error.connect(self._on_throttled_delete_error)
+        self.throttled_deleter.finished.connect(self._on_throttled_delete_finished)
+        self.throttled_deleter.transfer()
 
         # in case all the files to delete were local only
         if self.delete_files_finished == len(self._files_to_delete):
             self.delete_finished.emit()
 
+    def _on_throttled_delete_error(self, filename: str, error: str) -> None:
```

Comment:
Do we do anything with the error?

Reply:
we do, it is used via line 715 in `ThrottledFileTransferrer._on_delete_finished_wrapper`